README.md

```diff
@@ -1,20 +1,66 @@
 node-casbin
 ====
-
+[![NPM version][npm-image]][npm-url]
+[![NPM download][download-image]][download-url]
 [![codebeat badge](https://codebeat.co/badges/c17c9ee1-da42-4db3-8047-9574ad2b23b1)](https://codebeat.co/projects/github-com-casbin-node-casbin-master)
 [![Build Status](https://travis-ci.org/casbin/node-casbin.svg?branch=master)](https://travis-ci.org/casbin/node-casbin)
 [![Coverage Status](https://coveralls.io/repos/github/casbin/node-casbin/badge.svg?branch=master)](https://coveralls.io/github/casbin/node-casbin?branch=master)
 [![Release](https://img.shields.io/github/release/casbin/node-casbin.svg)](https://github.com/casbin/node-casbin/releases/latest)
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/casbin/lobby)
 [![Patreon](https://img.shields.io/badge/patreon-donate-yellow.svg)](http://www.patreon.com/yangluo)
-[![Sourcegraph Badge](https://sourcegraph.com/github.com/casbin/casbin/-/badge.svg)](https://sourcegraph.com/github.com/casbin/casbin?badge)
+[![Sourcegraph Badge](https://sourcegraph.com/github.com/casbin/node-casbin/-/badge.svg)](https://sourcegraph.com/github.com/casbin/node-casbin?badge)
+
+[npm-image]: https://img.shields.io/npm/v/casbin.svg?style=flat-square
+[npm-url]: https://npmjs.org/package/casbin
+[download-image]: https://img.shields.io/npm/dm/casbin.svg?style=flat-square
+[download-url]: https://npmjs.org/package/casbin
 
 **News**: still worry about how to write the correct node-casbin policy? ``Casbin online editor`` is coming to help! Try it at: http://casbin.org/editor/
 
 ![casbin Logo](casbin-logo.png)
 
 node-casbin is a powerful and efficient open-source access control library for Node.JS projects. It provides support for enforcing authorization based on various [access control models](https://en.wikipedia.org/wiki/Computer_security_model).
 
+## Installation
```

Comment:
I don't think it's appropriate to move ToC (table of contents) behind other contents. ToC is just something in the beginning and try to show readers which parts are where.